MetaMask/metamask-extension
# Various NFT UI fixes

## Explanation
Fixes: https://github.com/MetaMask/metamask-extension/issues/17148
Fixes: https://github.com/MetaMask/metamask-extension/issues/17144
Fixes: https://github.com/MetaMask/metamask-extension/issues/17143
Fixes: https://github.com/MetaMask/metamask-extension/issues/17139

Additional adjustments outside of tagged issues:

- A copy button has been added for the data URI in the NFT detail view, alignment of copy buttons fixed
- As a result of fixing the info icon alignment issue in the 'Import NFT' screen, this issue has also been fixed in several other areas throughout the UI that were using `<FormField/>` with an info tooltip.

## Screenshots

(https://github.com/MetaMask/metamask-extension/issues/17139):
<img width="449" alt="Screen Shot 2023-01-11 at 3 50 39 PM" src="https://user-images.githubusercontent.com/8732757/211985457-bd00423d-b493-4313-84c3-f9152cddddcb.png">

(https://github.com/MetaMask/metamask-extension/issues/17144):
<img width="531" alt="Screen Shot 2023-01-13 at 3 23 29 PM" src="https://user-images.githubusercontent.com/8732757/212430536-289e905b-b3e4-4504-b545-ef52a4921174.png">
<img width="530" alt="Screen Shot 2023-01-13 at 3 23 33 PM" src="https://user-images.githubusercontent.com/8732757/212430705-5732f5ed-4f40-4654-ae77-8a204bdf61c8.png">
<img width="525" alt="Screen Shot 2023-01-13 at 3 23 09 PM" src="https://user-images.githubusercontent.com/8732757/212430582-771deb05-de3c-4def-bfc5-ac41256e0e53.png">
<img width="527" alt="Screen Shot 2023-01-13 at 3 23 16 PM" src="https://user-images.githubusercontent.com/8732757/212430599-69398ab8-51c6-4524-8504-b70d18df5149.png">



(https://github.com/MetaMask/metamask-extension/issues/17148):
<img width="405" alt="Screen Shot 2023-01-11 at 10 28 49 PM" src="https://user-images.githubusercontent.com/8732757/211986198-81611d2c-fb0d-4a77-9b96-1cea64a4359c.png">
<img width="362" alt="Screen Shot 2023-01-11 at 10 29 03 PM" src="https://user-images.githubusercontent.com/8732757/211986200-be1e02b5-5ff9-401a-89b7-8fc0fd15fcad.png">
<img width="345" alt="Screen Shot 2023-01-11 at 10 29 28 PM" src="https://user-images.githubusercontent.com/8732757/211986203-cf62f41f-c4c1-43dc-b104-70c7ecdfb04c.png">

## Manual Testing Steps
1. Verify the static UI fixes in the given screens
2. On the NFT's tab, ensure that you can tab through the NFT thumbnails and access their respective detail pages via the keyboard.

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
